src/main/jbake/content/blog/2016/10/06/hawkular-metrics-0.20.0.Final-released.md

```diff
@@ -0,0 +1,102 @@
+- hm_s: string
+
+
+**Compression**
+
+Hawkular Metrics will now compress all the data with a compression scheme based on Facebook's Gorilla paper. A job will be run every two hours that compresses data received during the 2 hour block since last compression (there's one hour time for out-of-order writes to be written to the Cassandra). These data points are then written to a new storage format that's designed after the Gorilla compression and also compressed with the LZ4 algorithm to give even better compression ratio. Benefits include a faster reading time of larger block of datapoints and very large disk space savings. For more details: [HWKMETRICS-464](https://issues.jboss.org/browse/HWKMETRICS-464)
+
+
+**Other major changes**
+
+* REST API - Query Improvements
+   * Querying raw & rate data by tags is now also possible via POST `{metric_type}/query` endpoints for all types of metrics ([HWKMETRICS-466](https://issues.jboss.org/browse/HWKMETRICS-466))
+    * Added new POST endpoints for stats `gauges/stats/query` and `counters/stats/query` ([HWKMETRICS-465](https://issues.jboss.org/browse/HWKMETRICS-465))
+    * It is now possible to fetch metric definitions filtered by a list of ids not just tags ([HWKMETRICS-461](https://issues.jboss.org/browse/HWKMETRICS-461))
+
+* Cassandra Driver Configuration
+  * Driver configuration options are now exposed for connection and request timeouts
+  * Connection timeout can be configured via `CASSANDRA_CONNECTION_TIMEOUT` environment variable, or `hawkular-metrics.cassandra.connection.timeout` system property; value is specified in milliseconds and defaults to 5 seconds.
+  * Request timeout can be configured via `CASSANDRA_REQUEST_TIMEOUT` environment variable, or `hawkular-metrics.cassandra.request.timeout` system property; value is specified in milliseconds and defaults to 12 seconds.
+  * For more details: ([HWKMETRICS-490](https://issues.jboss.org/browse/HWKMETRICS-490))
+
+* Job Scheduler
+  * Duplicate instances of jobs were wrongly scheduled on server restart because the job scheduler was not checking if the job was already scheduled ([HWKMETRICS-461](https://issues.jboss.org/browse/HWKMETRICS-461))
+  * Triggers now have a delay to properly schedule jobs in the future ([HWKMETRICS-224](https://issues.jboss.org/browse/HWKMETRICS-224))
+  * Resolved an issue where long running job can miss future executions because the current execution takes longer than the repeat interval ([HWKMETRICS-477](https://issues.jboss.org/browse/HWKMETRICS-477))
+  * Prevent execution of a job multiple times for the same execution time ([HWKMETRICS-486](https://issues.jboss.org/browse/HWKMETRICS-486))
+  * RepeatingTrigger now allows specifying trigger start time ([HWKMETRICS-476](https://issues.jboss.org/browse/HWKMETRICS-476))
+
+* Admin Endpoints
+  * Admin related endpoints are going to be protected via an admin token, to be sent for admin related REST endpoints via Hawkular-Admin-Token request header
```

Comment:
Working on fixing this. It's really confusing because it rendered differently on Github.